### PR TITLE
[MRG+1] add plot_object_source_estimate.py to documentation

### PR DESCRIPTION
**Pull request by agramfort**
taking progressively pieces from source morph project.

### CODE REVIEW

**Comment by agramfort** [comment]
see rendered tutos:

https://9092-1301584-gh.circle-artifacts.com/0/html/auto_tutorials/plot_object_source_estimate.html
https://9092-1301584-gh.circle-artifacts.com/0/html/auto_tutorials/plot_background_freesurfer.html

**Comment by agramfort** [comment]
updated files:

https://9095-1301584-gh.circle-artifacts.com/0/html/auto_tutorials/plot_object_source_estimate.html#sphx-glr-auto-tutorials-plot-object-source-estimate-py

https://9095-1301584-gh.circle-artifacts.com/0/html/auto_tutorials/plot_background_freesurfer.html#sphx-glr-auto-tutorials-plot-background-freesurfer-py

@TommyClausner can you have a look?

**Comment by TommyClausner** [comment]
Hey looks good! ;)

Would you think it's a good idea to just add a short ``(shown here)`` or so to the ``(surface) SourceEstimate`` when listing the different types? (Just to emphasize that the user wont get info about Vol / Mix)?

Other than that: :)

File: tutorials/plot_object_source_estimate.py [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+    - :class:`mne.SourceEstimate`: For cortically constrained source spaces.
+
+    - :class:`mne.VolSourceEstimate`: For volumetric source spaces
+
+    - :class:`mne.VectorSourceEstimate`: For cortically constrained source
+      spaces with vector-valued source activations (strength and orientation)
+
+    - :class:`mne.MixedSourceEstimate`: For source spaces formed of a
+      combination of cortically constrained and volumetric sources.
+
```
Comment (TommyClausner) [inline tutorials/plot_object_source_estimate.py]: osd was hitting me so: some of the list entries have a period and others don't + I would add something like "...cortically constrained source spaces (demonstrated here)"

**Comment by agramfort** [comment]
ok thx @massich and @TommyClausner for the review. I'll merge this as is and invite you guys to submit a PR to make it even greater. I'll get back to source morphing now and rebase it on top of master.

**Pull request merged**